files/scripts/write_standby.py

```diff
@@ -96,14 +96,21 @@ def is_warmrestart(self):
 
         return status and value == 'true' 
 
-    def get_auto_mux_intfs(self):
+    def get_all_mux_intfs_modes(self):
         """
-        Returns a list of all mux cable interfaces that are configured to auto-switch
+        Returns a list of all mux cable interfaces, with suggested modes
+        Setting mux initial modes is crucial to kick off the statemachines,
+        have to set the modes for all mux/gRPC ports.
         """
+        intf_modes = {}
         all_intfs = self.config_db.get_table('MUX_CABLE')
-        auto_intfs = [intf for intf, status in all_intfs.items()
-                                if status['state'].lower() == 'auto']
-        return auto_intfs
+        for intf, status in all_intfs.items():
+            state = status['state'].lower()
+            if state == 'standby':
+                intf_modes[intf] = 'standby'
+            elif state in ['active', 'auto', 'manual']:
+                intf_modes[intf] = 'active'
```

Comment:
how does this behave for y-cable dual-tor?

Reply:
updated. thanks.